sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -750,9 +760,35 @@ object Radar {
             return
         }
 
+        RadarSettings.setListenToServerTrackingOptions(context, false)
         locationManager.startTracking(options)
     }
 
+    /**
+     * Starts tracking the user's location in the background using Radar dashboard settings.
+     *
+     * @see [](https://radar.io/documentation/sdk/android#background-tracking-for-geofencing)
+     */
+    @JvmStatic
+    fun startTracking() {
```

Comment:
1. make remote tracking more explicit startTracking(remote=true [via dashboard]), startTrackingRemote() + fallback = Responsive
2. far in the future, this will all be abstracted away in Radar.initialize

Reply:
3. factor out the foregroundService for trackingOptions to be client-specific